clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -5126,6 +5126,14 @@ void CXXNameMangler::mangleExpression(const Expr *E, unsigned Arity,
       Diags.Report(DiagID);
       return;
     }
+    case UETT_VectorElements: {
```

Comment:
We probably need the equivilent here for the MicrosoftMangle.cpp as well?

Reply:
I don't know. But none of the other `UETT_*` types are covered in `MicrosoftMangle.cpp`. I'm not sure whether they are missing or if it not necessary to add this.

Reply:
OK, SGTM at least, I just know this pattern of 'cannot yet mangle...' shows up primarily in the MicrosoftMangle, so figured it might need to be covered.  There is some funny-business as to how much each mangles of expressions in template arguments.

This gets me thinking further though,the constexprness of this likely means you may need to mangle this.  I'm open if others are to letting it be done in a follow-up however.

Reply:
@philnik777 I'm adding you here because of your recent commit that adds `__datasizeof` (#67805). This is essentially the same code path and you also specify the "cannot mangle" for Itanium but not for Microsoft. Do you know if this is needed for Microsoft (see comments above)? 

I'm happy to address this for `__builtin_vectorelements`, but I really don't know where to start and `__datasizeof` seems to be the same.